app/gui/qt/CMakeLists.txt

```
@@ -30,13 +30,20 @@ endif()
 # Qt Setup
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
 
 set(QT_TOOLS_DIR ${CMAKE_PREFIX_PATH}/bin)
 
 set_property(GLOBAL PROPERTY AUTOMOC_FOLDER Automoc)
 set_property(GLOBAL PROPERTY AUTOGEN_TARGETS_FOLDER Automoc)
 
-find_package(Qt5 COMPONENTS Core Widgets Gui Concurrent Network OpenGL PrintSupport Xml Svg WebEngineWidgets REQUIRED)
+
+find_package(Qt6 COMPONENTS Widgets Core Widgets Gui Concurrent Network OpenGLWidgets PrintSupport Xml Svg WebEngineWidgets REQUIRED)
```

It looks like `Widgets` got specified twice. That's not necessary, right?

```suggestion
find_package(Qt6 COMPONENTS Core Widgets Gui Concurrent Network OpenGLWidgets PrintSupport Xml Svg WebEngineWidgets REQUIRED)
```

---

haha, no!